README.md

```diff
@@ -1,17 +1,78 @@
+# Open Virtual Networking (OVN)
 
-This document describes how to use Open Virtual Networking with Kubernetes
+OVN provides network virtualization to containers.  In the "overlay" mode,
+OVN can create a logical network amongst containers running on multiple hosts.
+In this mode, OVN programs the Open vSwitch instances running inside your
+hosts.  These hosts can be bare-metal machines or vanilla VMs.
+
+## OVN via kubernetes daemonsets
+
+The easiest way to use OVN as the networking plugin (via CNI) for kubernetes
+is via kubernetes daemonsets. This currently only works for Linux nodes.
+
+One simply needs to apply the following yamls.
+
+1. Create a namespace `ovn-kubernetes` in kubernetes
+
+`kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/ovn-namespace.yaml`
+
+2. Create service accounts and policies for OVN
+
+`kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/ovn-policy.yaml`
+
+3. Run a config map file for OVN.
+
+* $MASTER_IP below is the IP address of the
+kubernetes master node.
+
+* A cluster wide private address range of $CLUSTER_IP_SUBNET
+(e.g: 192.168.0.0/16).  The pods are provided IP address from this range.
+
+* $SERVICE_IP_SUBNET is the same as the one provided to k8s-apiserver via
+--service-cluster-ip-range option. An e.g is 172.16.1.0/24.
+
+```
+cat << EOF | kubectl create -f - > /dev/null 2>&1
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovn-config
+  namespace: ovn-kubernetes
+data:
+  k8s_apiserver: "https://$MASTER_IP:6443"
+  net_cidr:      $CLUSTER_IP_SUBNET
+  svc_cidr:      $SERVICE_IP_SUBNET
+  OvnNorth:      tcp://$MASTER_IP:6641
+  OvnSouth:      tcp://$MASTER_IP:6642
+EOF
+```
+
+4. Label the master node for daemonsets
+
+The below command needs $MASTER_NODE_NAME to be populated.
+
+`kubectl label node $MASTER_NODE_NAME node-role.kubernetes.io/master=true --overwrite`
+
+5. Run OVN daemonsets.
+
+```
+# Run ovnkube-master daemonset.
+kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/static/ovnkube-master.yaml
+# Run ovnkube daemonsets for nodes
+kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/static/ovnkube.yaml
+```
+
+That is all.
+
+## How to use Open Virtual Networking with Kubernetes for advanced users.
+
+The following describes how to use Open Virtual Networking with Kubernetes
 1.8.0 or later.  This document assumes that you have installed Open
 vSwitch by following [INSTALL.rst] or by using the distribution packages
 such as .deb or.rpm.
 
 ## Setup
 
-OVN provides network virtualization to containers.  In the "overlay" mode,
-OVN can create a logical network amongst containers running on multiple hosts.
-In this mode, OVN programs the Open vSwitch instances running inside your
-hosts.  These hosts can be bare-metal machines or vanilla VMs.
-
 Installing Open vSwitch is out of scope of this documentation.  You can read
```

Comment:
In the setup section, there is a discussion of installing ovs. With daemonsets we don't install ovn on the hosts, its in the image and all ovn commands are run from within containers.

Reply:
That is for non-daemonset part. It is in a separate section (for advanced users)